config/README.md

```diff
@@ -108,7 +108,7 @@ redis:
   # list of addresses to redis nodes
   # you should use multiple addresses only if they all belong to the same redis cluster.
   addresses:
-    - <string> # example "localhost:6379"
+    - <string> # example "localhost:16379"
```

Comment:
you should not change this port since it's the doc and IHMO it's better to keep the default port of redis

Reply:
@mga-chka you are right. I fixed it